.github/workflows/assign-reviewers.yml

```diff
@@ -8,7 +8,7 @@ jobs:
   # JSON files in codeowners/.
   request_reviewer:
     name: "Request review from code owner"
-    runs-on: ubuntu-18.04
```

Comment:
FYI: this workflow broke a while back and it wasn't useful enough for me to fix so I just disabled it. It should just be deleted and CODEOWNERs should be put in a readable format.

Reply:
Not requesting any changes.

Reply:
Ah fair enough, just did a find-and-replace here, not worrying about it.